mmv1/third_party/terraform/.teamcity/components/inputs/services_ga.kt

```diff
@@ -611,6 +611,11 @@ var ServicesListGa = mapOf(
         "displayName" to "Secretmanager",
         "path" to "./google/services/secretmanager"
     ),
+    "secretmanagerregional" to mapOf(
+        "name" to "secretmanagerregional",
+        "displayName" to "Secretmanagerregional",
```

Comment:
Same as beta.

Reply:
The Action for teamcity was failing regarding this. After I've added this snippet, the workflow was failing: https://github.com/GoogleCloudPlatform/magic-modules/actions/runs/10806885008/job/29976554704. I am not sure if we need to add this snippet.

Reply:
Verified, existing resources like IAM are also adding this.